### PR TITLE
Kepler-ifying Part 2

### PR DESCRIPTION
**Pull request by AffectedArc07**
## About The Pull Request
This PR guts out most of the lewd aspects from the game, mainly vore and sex organs.

## Why It's Good For The Game
We dont want this degenerate trash

## Changelog
:cl: AffectedArc07
del: Removed ERP
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Bitch bot

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -875,11 +875,6 @@
 	attack_verb = list("headbutt", "scritched", "bit")
 	squeak_override = list('modular_citadel/sound/voice/nya.ogg' = 1)
 
-/obj/item/toy/plush/catgirl/mikeel
```
Comment (CornMyCob) [inline code/game/objects/items/plushes.dm]: Rather you delete one or all of them.

File: modular_citadel/code/_onclick/hud/screen_objects.dm [hunk 2]
```diff
@@ -16,7 +16,7 @@
 	if(!H)
 		return
 	if(H.sprinting)
-		icon_state = "act_sprint_on"
+		icon_state = "act_sprint_own"
```
Comment (CornMyCob) [inline modular_citadel/code/_onclick/hud/screen_objects.dm]: Mistake?

File: code/game/objects/items/plushes.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/game/objects/items/plushes.dm]: Ill bring back, probably

File: modular_citadel/code/_onclick/hud/screen_objects.dm (hunk 2 above)
Comment (AffectedArc07) [reply modular_citadel/code/_onclick/hud/screen_objects.dm]: Oops

**Pull request merged**